settings/common.py

```diff
@@ -273,3 +273,8 @@
     },
     'SUBMISSION_QUEUE': 'submission_task_queue',
 }
+
+# The maximum size in bytes for request body
+# https://docs.djangoproject.com/en/1.10/ref/settings/#data-upload-max-memory-size
+FILE_UPLOAD_MAX_MEMORY_SIZE = 10485760  # 10 MB
+DATA_UPLOAD_MAX_MEMORY_SIZE = 10485760  # 10 MB
```

Comment:
@memclutter Just a small change, please change the limit to `500 MB`.

Reply:
@RishabhJain2018 : 500 mb is too much? Also you will need to increase the limit at Nginx as well

@deshraj

Reply:
@taranjeet I know, but challenge file can be of `500 MB` as @deshraj suggested. We can increase the limit at nginx too.